projects/libs/core-library-typescript/src/common-library/startup-status.ts

```diff
@@ -0,0 +1,14 @@
+export interface StartupStatus {
+  name: string;
+  status: STARTUP_STATUS;
+  childStartupStatuses?: StartupStatus[];
+  message?: string;
+}
+
+// export interface CompositeStartupStatus {
```

Comment:
we don't need this? (its okay if left for future use, but in general no-one likes commented code, its evil)